pkgs/development/python-modules/django-timezone-field/default.nix

```diff
@@ -13,15 +13,13 @@
 
 buildPythonPackage rec {
   pname = "django-timezone-field";
-  version = "4.2.3";
+  version = "5.0";
   format = "setuptools";
   disabled = pythonOlder "3.5";
 
-  src = fetchFromGitHub {
-    owner = "mfogel";
-    repo = pname;
-    rev = version;
-    sha256 = "sha256-purzXzPvnYNVsASsK6s9m4XieUL80FOjeSbUP4bpRj8=";
+  src = fetchPypi {
```

Comment:
Why do we want to fetch from pypi? Normally GitHub contains all sources including tests and the sdist can easily be altered which is bad for security and makes applying patches harder.